FAQ.md

```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
 Not really. For instance, the demonstration server [https://peertube.cpy.re](https://peertube.cpy.re) has 2 vCore and 2GB of RAM and consumes on average:
  * **CPU** -> nginx ~ 20%, peertube ~ 10%,   postgres ~ 1%, redis ~ 3%
  * **RAM** -> nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
- 
+
 So you would need:
  * **CPU** 1 core if you don't enable transcoding, 2 at least if you enable it
  * **RAM** 1GB
  * **Storage** Completely depends on how many videos your users will upload
 
+
+## Are you gonna use the Steem blockchain?
+
+Short answer: no, since like most appchains/votechains, it modifies the dynamic of creation, and as such cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
+
+Long answer is that the Steem blockchain goes astray of its promises of fairness and decentralization: the deliberate relaunching of the currency to ensure centralization, and the stake-based voting power, makes manipulation by wealthy users inevitable ([source here](https://decentralize.today/the-ugly-truth-behind-steemit-1a525f5e156)). Worse, money generated goes primarily to stake holders ([source here](https://steemit.com/steemit/@orly/how-the-steem-pyramid-scheme-really-works) ). For more information, read the complete whitepaper analysis done by [Tone Vays](https://twitter.com/ToneVays/status/761975587451928576).
+
+## Are you gonna support adding ads?
+
+Short answer: no, we don't want advertisers to dictate which content should be financed. That would modify the dynamic of creation ; as such it cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
+
+The long answer is probably more subtle. YouTube has shaped generations of video creators by making it easy to place ads ; but making big money with the platform can be a challenge. A typical video ad runs between $.10 and $.30 per 1000 view (as of march 2018). More than 70% of video creators have ads as their main way to make money on YouTube, yet less than 3% of video creators make a living out of their YouTube activity (with partnerships and commissions, otherwise counting only ad revenue it drops to 1%). Read more about it in the 2018 study by Mathias Bärtl, [*YouTube channels, uploads and views: A statistical analysis of the past 10 years*](https://www.dropbox.com/s/0cq4wtxm83s95t2/10.1177%401354856517736979.pdf?dl=0). To the best of our knowledge, small and medium-community creators are better off getting support from their community on platforms such as Liberapay, Tipeee or Patreon. Moreover, don't forget that advertisers already pay considering YouTube's large user base ; with PeerTube's way smaller user base and refusal of user profiling, a lower pay-per-view than on YouTube could only be expected.
```

Comment:
same typo nit again a few times here and below (no space before semicolon).

nit: `per 1000 views`
nit: `as of March 2018`
suggestion: `have ads as their main way` => `use ads as the main way`
suggestion: `a lower pay-per-view than on YouTube` => `a pay-per-view that's lower than YouTube's`

Reply:
I corrected that, thanks! Your suggestions are less convoluted than mine, I'll use them.